From: Miguel Telleria de Esteban <miguel@mtelleria.com>
Date: Thu, 14 Jun 2012 11:58:53 +0200
Subject: build.sh must create tds directory before cp-r biblatex

---
 build/build.sh |    4 ++++
 1 file changed, 4 insertions(+)

Index: biber/build/build.sh
===================================================================
--- biber.orig/build/build.sh	2012-06-14 13:02:20.000000000 +0200
+++ biber/build/build.sh	2012-06-14 13:06:21.000000000 +0200
@@ -20,6 +20,7 @@
 
 \rm -rf build/tds/*
 \rm -f build/biblatex.tgz
+mkdir build/tds
 cp -r bibtex build/tds/
 cp -r doc build/tds/
 cp -r tex build/tds/
